OpenSUSE Dockerfile does not build:  `chpasswd` command not found

When releasing the v0.10.0 images, our OpenSUSE Leap OS image build [failed](https://github.com/weaveworks/ignite/runs/3107948135).
The root of the problem may be that upstream SUSE removed `chpasswd` from their base container image.
The solution is hopefully simple: find the package(s) containing missing binaries so that we can make the build succeed again.

This is how you can build the image locally:
```
cd repos/ignite
cd images/opensuse
make DOCKER_USER=$USER build-opensuse
```

For now, the v0.10.0 image will remain unreleased.
__________________
<details>
<summary>Failed Build Log</summary>

```
Run make OP=build build-opensuse
make build WHAT=opensuse       RELEASE=leap                                     IS_LATEST=true
make[1]: Entering directory '/home/runner/work/ignite/ignite/images'
sed "s|DOCKERARCH|amd64|g;/QEMUARCH/d" opensuse/Dockerfile \
	| docker build \
		--build-arg RELEASE \
		--build-arg BINARY_REF \
		--build-arg GOARCH \
		-f -\
		-t weaveworks/ignite-opensuse:leap-v0.10.0 \
		opensuse
Sending build context to Docker daemon  2.607kB

Step 1/5 : ARG RELEASE
Step 2/5 : FROM opensuse/${RELEASE}
latest: Pulling from opensuse/leap
b79179246e29: Pulling fs layer
b79179246e29: Verifying Checksum
b79179246e29: Download complete
b79179246e29: Pull complete
Digest: sha256:5ef58a5bbefa7957d92ecfad26e7c2b54f453c8322dc8b074d578a55d3760eef
Status: Downloaded newer image for opensuse/leap:latest
 ---> 25fd0a4269ec
Step 3/5 : RUN zypper -n install         iproute         iputils         openssh         net-tools         systemd-sysvinit         udev         sudo         wget &&     zypper clean --all
 ---> Running in 268efd201ed6
Retrieving repository 'Update repository of openSUSE Backports' metadata [...done]
Building repository 'Update repository of openSUSE Backports' cache [....done]
Retrieving repository 'Non-OSS Repository' metadata [...done]
Building repository 'Non-OSS Repository' cache [....done]
Retrieving repository 'Main Repository' metadata [......done]
Building repository 'Main Repository' cache [....done]
Retrieving repository 'Update repository with updates from SUSE Linux Enterprise 15' metadata [..................done]
Building repository 'Update repository with updates from SUSE Linux Enterprise 15' cache [....done]
Retrieving repository 'Main Update Repository' metadata [...done]
Building repository 'Main Update Repository' cache [....done]
Retrieving repository 'Update Repository (Non-Oss)' metadata [...done]
Building repository 'Update Repository (Non-Oss)' cache [....done]
Loading repository data...
Reading installed packages...
'iproute' not found in package names. Trying capabilities.
Resolving package dependencies...

The following 46 NEW packages are going to be installed:
  dbus-1 gzip hostname iproute2 iputils kbd kbd-legacy kmod libapparmor1 libargon2-1 libcares2 libcbor0 libcryptsetup12 libdbus-1-3 libdevmapper1_03 libedit0 libexpat1 libfido2-1 libfido2-udev libidn11 libjson-c3 libkmod2 libmetalink3 libmnl0 libpcre2-8-0 libqrencode4 libseccomp2 libxtables12 net-tools openssh openssh-clients openssh-common openssh-server pam-config pkg-config sudo suse-module-tools system-group-kvm systemd systemd-default-settings systemd-default-settings-branding-SLE systemd-presets-branding-MicroOS systemd-presets-common-SUSE systemd-sysvinit udev wget

46 new packages to install.
Overall download size: 13.8 MiB. Already cached: 0 B. After the operation, additional 43.6 MiB will be used.
Continue? [y/n/v/...? shows all options] (y): y
Retrieving package hostname-3.16-2.22.x86_64 (1/46),  22.8 KiB ( 34.5 KiB unpacked)
Retrieving: hostname-3.16-2.22.x86_64.rpm [done]
Retrieving package kbd-legacy-2.0.4-14.38.noarch (2/46), 495.7 KiB (517.0 KiB unpacked)
Retrieving: kbd-legacy-2.0.4-14.38.noarch.rpm [done]
Retrieving package libapparmor1-2.13.6-1.24.x86_64 (3/46),  39.2 KiB ( 75.6 KiB unpacked)
Retrieving: libapparmor1-2.13.6-1.24.x86_64.rpm [.done (15.5 KiB/s)]
Retrieving package libargon2-1-0.0+git20171227.670229c-2.14.x86_64 (4/46),  19.9 KiB ( 30.2 KiB unpacked)
Retrieving: libargon2-1-0.0+git20171227.670229c-2.14.x86_64.rpm [done]
Retrieving package libcares2-1.17.0-3.11.1.x86_64 (5/46),  54.9 KiB ( 90.7 KiB unpacked)
Retrieving: libcares2-1.17.0-3.11.1.x86_64.rpm [done]
Retrieving package libcbor0-0.5.0-2.38.x86_64 (6/46),  47.0 KiB ( 85.3 KiB unpacked)
Retrieving: libcbor0-0.5.0-2.38.x86_64.rpm [done]
Retrieving package libedit0-3.1.snap20150325-2.12.x86_64 (7/46),  94.3 KiB (222.2 KiB unpacked)
Retrieving: libedit0-3.1.snap20150325-2.12.x86_64.rpm [done]
Retrieving package libexpat1-2.2.5-3.6.1.x86_64 (8/46),  75.1 KiB (198.3 KiB unpacked)
Retrieving: libexpat1-2.2.5-3.6.1.x86_64.rpm [done]
Retrieving package libfido2-udev-1.5.0-1.30.noarch (9/46),  13.8 KiB (  8.2 KiB unpacked)
Retrieving: libfido2-udev-1.5.0-1.30.noarch.rpm [.done (15.5 KiB/s)]
Retrieving package libidn11-1.34-3.2.2.x86_64 (10/46),  72.8 KiB (293.2 KiB unpacked)
Retrieving: libidn11-1.34-3.2.2.x86_64.rpm [done]
Retrieving package libjson-c3-0.13-1.19.x86_64 (11/46),  40.8 KiB ( 64.9 KiB unpacked)
Retrieving: libjson-c3-0.13-1.19.x86_64.rpm [done]
Retrieving package libkmod2-28-2.1.x86_64 (12/46),  54.6 KiB ( 98.1 KiB unpacked)
Retrieving: libkmod2-28-2.1.x86_64.rpm [done]
Retrieving package libmnl0-1.0.4-1.25.x86_64 (13/46),  16.4 KiB ( 22.4 KiB unpacked)
Retrieving: libmnl0-1.0.4-1.25.x86_64.rpm [done]
Retrieving package libpcre2-8-0-10.31-1.14.x86_64 (14/46), 235.5 KiB (648.0 KiB unpacked)
Retrieving: libpcre2-8-0-10.31-1.14.x86_64.rpm [done]
Retrieving package libqrencode4-4.0.0-1.17.x86_64 (15/46),  28.8 KiB ( 50.0 KiB unpacked)
Retrieving: libqrencode4-4.0.0-1.17.x86_64.rpm [.done (15.5 KiB/s)]
Retrieving package libseccomp2-2.4.1-3.3.1.x86_64 (16/46),  55.6 KiB (306.1 KiB unpacked)
Retrieving: libseccomp2-2.4.1-3.3.1.x86_64.rpm [done]
Retrieving package libxtables12-1.8.7-1.1.x86_64 (17/46),  34.3 KiB ( 58.9 KiB unpacked)
Retrieving: libxtables12-1.8.7-1.1.x86_64.rpm [done]
Retrieving package pam-config-1.1-1.43.x86_64 (18/46), 127.6 KiB (635.6 KiB unpacked)
Retrieving: pam-config-1.1-1.43.x86_64.rpm [done]
Retrieving package pkg-config-0.29.2-1.436.x86_64 (19/46), 231.6 KiB (675.4 KiB unpacked)
Retrieving: pkg-config-0.29.2-1.436.x86_64.rpm [done]
Retrieving package sudo-1.9.5p2-1.5.x86_64 (20/46),   1.1 MiB (  4.4 MiB unpacked)
Retrieving: sudo-1.9.5p2-1.5.x86_64.rpm [.done (1.8 MiB/s)]
Retrieving package system-group-kvm-20170617-15.86.noarch (21/46),  11.3 KiB (   37   B unpacked)
Retrieving: system-group-kvm-20170617-15.86.noarch.rpm [done]
Retrieving package systemd-default-settings-branding-SLE-0.7-1.1.noarch (22/46),  10.1 KiB (  1.5 KiB unpacked)
Retrieving: systemd-default-settings-branding-SLE-0.7-1.1.noarch.rpm [done]
Retrieving package net-tools-2.0+git20170221.479bb4a-3.11.x86_64 (23/46),  62.9 KiB (117.7 KiB unpacked)
Retrieving: net-tools-2.0+git20170221.479bb4a-3.11.x86_64.rpm [done]
Retrieving package kbd-2.0.4-14.38.x86_64 (24/46),   1.8 MiB (  4.2 MiB unpacked)
Retrieving: kbd-2.0.4-14.38.x86_64.rpm [done]
Retrieving package libmetalink3-0.1.3-1.24.x86_64 (25/46),  27.5 KiB ( 68.1 KiB unpacked)
Retrieving: libmetalink3-0.1.3-1.24.x86_64.rpm [.done (13.6 KiB/s)]
Retrieving package libfido2-1-1.5.0-1.30.x86_64 (26/46),  62.6 KiB (150.4 KiB unpacked)
Retrieving: libfido2-1-1.5.0-1.30.x86_64.rpm [done]
Retrieving package iputils-s20161105-8.3.1.x86_64 (27/46),  96.6 KiB (207.2 KiB unpacked)
Retrieving: iputils-s20161105-8.3.1.x86_64.rpm [done]
Retrieving package iproute2-5.3-5.2.1.x86_64 (28/46), 890.9 KiB (  2.5 MiB unpacked)
Retrieving: iproute2-5.3-5.2.1.x86_64.rpm [done]
Retrieving package systemd-default-settings-0.7-1.1.noarch (29/46),  11.2 KiB (  2.0 KiB unpacked)
Retrieving: systemd-default-settings-0.7-1.1.noarch.rpm [done]
Retrieving package gzip-1.10-7.1.x86_64 (30/46), 139.8 KiB (313.5 KiB unpacked)
Retrieving: gzip-1.10-7.1.x86_64.rpm [.done]
Retrieving package libdbus-1-3-1.12.2-8.6.1.x86_64 (31/46), 150.0 KiB (325.9 KiB unpacked)
Retrieving: libdbus-1-3-1.12.2-8.6.1.x86_64.rpm [done]
Retrieving package libdevmapper1_03-1.02.163-8.33.1.x86_64 (32/46), 174.1 KiB (349.8 KiB unpacked)
Retrieving: libdevmapper1_03-1.02.163-8.33.1.x86_64.rpm [done]
Retrieving package openssh-common-8.4p1-3.3.1.x86_64 (33/46), 459.2 KiB (  1.5 MiB unpacked)
Retrieving: openssh-common-8.4p1-3.3.1.x86_64.rpm [.done (924 B/s)]
Retrieving package systemd-presets-common-SUSE-15-8.9.1.noarch (34/46),  19.2 KiB (  4.3 KiB unpacked)
Retrieving: systemd-presets-common-SUSE-15-8.9.1.noarch.rpm [done]
Retrieving package wget-1.20.3-3.12.1.x86_64 (35/46), 386.9 KiB (767.9 KiB unpacked)
Retrieving: wget-1.20.3-3.12.1.x86_64.rpm [done]
Retrieving package suse-module-tools-15.3.8-3.5.1.x86_64 (36/46),  43.7 KiB ( 79.5 KiB unpacked)
Retrieving: suse-module-tools-15.3.8-3.5.1.x86_64.rpm [.done]
Retrieving package dbus-1-1.12.2-8.6.1.x86_64 (37/46), 248.8 KiB (624.3 KiB unpacked)
Retrieving: dbus-1-1.12.2-8.6.1.x86_64.rpm [done]
Retrieving package openssh-server-8.4p1-3.3.1.x86_64 (38/46), 513.1 KiB (  1.3 MiB unpacked)
Retrieving: openssh-server-8.4p1-3.3.1.x86_64.rpm [done]
Retrieving package openssh-clients-8.4p1-3.3.1.x86_64 (39/46), 707.6 KiB (  3.6 MiB unpacked)
Retrieving: openssh-clients-8.4p1-3.3.1.x86_64.rpm [.done (6.5 KiB/s)]
Retrieving package openssh-8.4p1-3.3.1.x86_64 (40/46), 108.9 KiB (    0   B unpacked)
Retrieving: openssh-8.4p1-3.3.1.x86_64.rpm [done]
Retrieving package libcryptsetup12-2.3.4-1.34.x86_64 (41/46), 217.3 KiB (461.6 KiB unpacked)
Retrieving: libcryptsetup12-2.3.4-1.34.x86_64.rpm [done]
Retrieving package systemd-presets-branding-MicroOS-20200323-lp153.2.14.noarch (42/46),  12.4 KiB (  695   B unpacked)
Retrieving: systemd-presets-branding-MicroOS-20200323-lp153.2.14.noarch.rpm [done]
Retrieving package kmod-28-2.1.x86_64 (43/46),  99.8 KiB (175.3 KiB unpacked)
Retrieving: kmod-28-2.1.x86_64.rpm [.done (15.5 KiB/s)]
Retrieving package systemd-246.13-7.5.1.x86_64 (44/46),   3.4 MiB ( 10.9 MiB unpacked)
Retrieving: systemd-246.13-7.5.1.x86_64.rpm [done]
Retrieving package udev-246.13-7.5.1.x86_64 (45/46),   1.3 MiB (  7.6 MiB unpacked)
Retrieving: udev-246.13-7.5.1.x86_64.rpm [.done (15.3 KiB/s)]
Retrieving package systemd-sysvinit-246.13-7.5.1.x86_64 (46/46), 195.4 KiB (  4.6 KiB unpacked)
Retrieving: systemd-sysvinit-246.13-7.5.1.x86_64.rpm [done]

Checking for file conflicts: [.........done]
( 1/46) Installing: hostname-3.16-2.22.x86_64 [......done]
( 2/46) Installing: kbd-legacy-2.0.4-14.38.noarch [............done]
( 3/46) Installing: libapparmor1-2.13.6-1.24.x86_64 [......done]
( 4/46) Installing: libargon2-1-0.0+git20171227.670229c-2.14.x86_64 [....done]
( 5/46) Installing: libcares2-1.17.0-3.11.1.x86_64 [......done]
( 6/46) Installing: libcbor0-0.5.0-2.38.x86_64 [......done]
( 7/46) Installing: libedit0-3.1.snap20150325-2.12.x86_64 [..........done]
( 8/46) Installing: libexpat1-2.2.5-3.6.1.x86_64 [.........done]
( 9/46) Installing: libfido2-udev-1.5.0-1.30.noarch [....done]
(10/46) Installing: libidn11-1.34-3.2.2.x86_64 [...........done]
(11/46) Installing: libjson-c3-0.13-1.19.x86_64 [.....done]
(12/46) Installing: libkmod2-28-2.1.x86_64 [......done]
(13/46) Installing: libmnl0-1.0.4-1.25.x86_64 [....done]
(14/46) Installing: libpcre2-8-0-10.31-1.14.x86_64 [..........done]
(15/46) Installing: libqrencode4-4.0.0-1.17.x86_64 [.....done]
(16/46) Installing: libseccomp2-2.4.1-3.3.1.x86_64 [........done]
(17/46) Installing: libxtables12-1.8.7-1.1.x86_64 [.....done]
(18/46) Installing: pam-config-1.1-1.43.x86_64 [...........done]
Additional rpm output:
*** write_config (account, /etc/pam.d/common-account-pc, ...)
**** write config for pam_access.so (account, disabled)
**** write config for pam_unix2.so (account, disabled)
**** write config for pam_unix.so (account, enabled)
**** write config for pam_krb5.so (account, disabled)
**** write config for pam_localuser.so (account, disabled)
**** write config for pam_sss.so (account, disabled)
**** write config for pam_ldap.so (account, disabled)
**** write config for pam_nam.so (account, disabled)
**** write config for pam_winbind.so (account, disabled)
**** write config for pam_time.so (account, disabled)
*** write_config (auth, /etc/pam.d/common-auth-pc, ...)
**** write config for pam_env.so (auth, enabled)
**** write config for pam_group.so (auth, disabled)
**** write config for pam_pkcs11.so (auth, disabled)
**** write config for pam_fp.so (auth, disabled)
**** write config for pam_fprint.so (auth, disabled)
**** write config for pam_fprintd.so (auth, disabled)
**** write config for pam_thinkfinger.so (auth, disabled)
**** write config for pam_gnome_keyring.so (auth, disabled)
**** write config for pam_kwallet5.so (auth, disabled)
**** write config for pam_ssh.so (auth, disabled)
**** write config for pam_unix2.so (auth, disabled)
**** write config for pam_unix.so (auth, enabled)
**** write config for pam_ecryptfs.so (auth, disabled)
**** write config for pam_krb5.so (auth, disabled)
**** write config for pam_sss.so (auth, disabled)
**** write config for pam_ldap.so (auth, disabled)
**** write config for pam_nam.so (auth, disabled)
**** write config for pam_winbind.so (auth, disabled)
*** write_config (password, /etc/pam.d/common-password-pc, ...)
**** write config for pam_winbind.so (password, disabled)
**** write config for pam_pwcheck.so (password, disabled)
**** write config for pam_passwdqc.so (password, disabled)
**** write config for pam_cracklib.so (password, enabled)
**** write config for pam_pwhistory.so (password, disabled)
**** write config for pam_gnome_keyring.so (password, disabled)
**** write config for pam_kwallet5.so (password, disabled)
**** write config for pam_ecryptfs.so (password, disabled)
**** write config for pam_unix2.so (password, disabled)
**** write config for pam_unix.so (password, enabled)
**** write config for pam_make.so (password, disabled)
**** write config for pam_exec.so (password, disabled)
**** write config for pam_krb5.so (password, disabled)
**** write config for pam_sss.so (password, disabled)
**** write config for pam_ldap.so (password, disabled)
**** write config for pam_nam.so (password, disabled)
*** write_config (session, /etc/pam.d/common-session-pc, ...)
**** write config for pam_selinux.so (session, disabled)
**** write config for pam_mkhomedir.so (session, disabled)
**** write config for pam_systemd.so (session, disabled)
**** write config for pam_limits.so (session, enabled)
**** write config for pam_unix2.so (session, disabled)
**** write config for pam_unix.so (session, enabled)
**** write config for pam_apparmor.so (session, disabled)
**** write config for pam_krb5.so (session, disabled)
**** write config for pam_sss.so (session, disabled)
**** write config for pam_ldap.so (session, disabled)
**** write config for pam_winbind.so (session, disabled)
**** write config for pam_nam.so (session, disabled)
**** write config for pam_umask.so (session, enabled)
**** write config for pam_ssh.so (session, disabled)
**** write config for pam_selinux.so (session, disabled)
**** write config for pam_gnome_keyring.so (session, disabled)
**** write config for pam_kwallet5.so (session, disabled)
**** write config for pam_exec.so (session, disabled)
**** write config for pam_ecryptfs.so (session, disabled)
**** write config for pam_env.so (session, enabled)
**** write config for pam_mktemp.so (session, disabled)


(19/46) Installing: pkg-config-0.29.2-1.436.x86_64 [...........done]
(20/46) Installing: sudo-1.9.5p2-1.5.x86_64 [............done]
(21/46) Installing: system-group-kvm-20170617-15.86.noarch [....done]
Additional rpm output:
groupadd -r -g 36 kvm


(22/46) Installing: systemd-default-settings-branding-SLE-0.7-1.1.noarch [........done]
(23/46) Installing: net-tools-2.0+git20170221.479bb4a-3.11.x86_64 [........done]
(24/46) Installing: kbd-2.0.4-14.38.x86_64 [............done]
Additional rpm output:
Updating /etc/sysconfig/console ...
Updating /etc/sysconfig/keyboard ...


(25/46) Installing: libmetalink3-0.1.3-1.24.x86_64 [.....done]
(26/46) Installing: libfido2-1-1.5.0-1.30.x86_64 [........done]
(27/46) Installing: iputils-s20161105-8.3.1.x86_64 [.........done]
Additional rpm output:
setting /usr/bin/ping to root:root 0755 "= cap_net_raw+ep". (wrong missing capabilities)


(28/46) Installing: iproute2-5.3-5.2.1.x86_64 [............done]
(29/46) Installing: systemd-default-settings-0.7-1.1.noarch [.........done]
(30/46) Installing: gzip-1.10-7.1.x86_64 [..........done]
(31/46) Installing: libdbus-1-3-1.12.2-8.6.1.x86_64 [.........done]
(32/46) Installing: libdevmapper1_03-1.02.163-8.33.1.x86_64 [.........done]
(33/46) Installing: openssh-common-8.4p1-3.3.1.x86_64 [............done]
(34/46) Installing: systemd-presets-common-SUSE-15-8.9.1.noarch [......done]
(35/46) Installing: wget-1.20.3-3.12.1.x86_64 [...........done]
(36/46) Installing: suse-module-tools-15.3.8-3.5.1.x86_64 [..........done]
(37/46) Installing: dbus-1-1.12.2-8.6.1.x86_64 [...........done]
Additional rpm output:
/var/tmp/rpm-tmp.btwrIU: line 34: [: : integer expression expected
update-alternatives: using /usr/bin/dbus-launch.nox11 to provide /usr/bin/dbus-launch (dbus-launch) in auto mode


(38/46) Installing: openssh-server-8.4p1-3.3.1.x86_64 [............done]
Additional rpm output:
find: '/etc/init.d/rc[35].d': No such file or directory
useradd -r -s /sbin/nologin -c "SSH daemon" -U -d /var/lib/sshd sshd
Updating /etc/sysconfig/ssh ...


(39/46) Installing: openssh-clients-8.4p1-3.3.1.x86_64 [............done]
(40/46) Installing: openssh-8.4p1-3.3.1.x86_64 [...done]
Additional rpm output:
find: '/etc/init.d/rc[35].d': No such file or directory


(41/46) Installing: libcryptsetup12-2.3.4-1.34.x86_64 [..........done]
(42/46) Installing: systemd-presets-branding-MicroOS-20200323-lp153.2.14.noarch [.....done]
(43/46) Installing: kmod-28-2.1.x86_64 [.........done]
(44/46) Installing: systemd-246.13-7.5.1.x86_64 [............done]
Additional rpm output:
Creating group systemd-journal with gid 485.
Creating group systemd-network with gid 484.
Creating user systemd-network (systemd Network Management) with uid 484 and gid 484.
Creating group systemd-resolve with gid 483.
Creating user systemd-resolve (systemd Resolver) with uid 483 and gid 483.
Creating group systemd-timesync with gid 482.
Creating user systemd-timesync (systemd Time Synchronization) with uid 482 and gid 482.
Creating group systemd-coredump with gid 481.
Creating user systemd-coredump (systemd Core Dumper) with uid 481 and gid 481.
System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down
/usr/lib/tmpfiles.d/journal-nocow.conf:26: Failed to resolve specifier: uninitialized /etc detected, skipping
All rules containing unresolvable specifiers will be skipped.
System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down
System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down
System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down
System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down
System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down


(45/46) Installing: udev-246.13-7.5.1.x86_64 [............done]
(46/46) Installing: systemd-sysvinit-246.13-7.5.1.x86_64 [..........done]
Executing %posttrans script 'kbd-2.0.4-14.38.x86_64.rpm' [..
Output of kbd-2.0.4-14.38.x86_64.rpm %posttrans script:
    Please run mkinitrd as soon as your system is complete.

.
Output of systemd-presets-common-SUSE-15-8.9.1.noarch.rpm %posttrans script:
    Created symlink /etc/systemd/system/default.target.wants/ca-certificates.path -> /usr/lib/systemd/system/ca-certificates.path.
    Created symlink /etc/systemd/system/getty.target.wants/getty@tty1.service -> /usr/lib/systemd/system/getty@.service.
    Created symlink /etc/systemd/system/multi-user.target.wants/remote-fs.target -> /usr/lib/systemd/system/remote-fs.target.
    Created symlink /etc/systemd/system/multi-user.target.wants/kbdsettings.service -> /usr/lib/systemd/system/kbdsettings.service.
    Created symlink /etc/systemd/system/multi-user.target.wants/sshd.service -> /usr/lib/systemd/system/sshd.service.
    Created symlink /etc/systemd/user/basic.target.wants/systemd-tmpfiles-setup.service -> /usr/lib/systemd/user/systemd-tmpfiles-setup.service.
    Created symlink /etc/systemd/user/timers.target.wants/systemd-tmpfiles-clean.timer -> /usr/lib/systemd/user/systemd-tmpfiles-clean.timer.

......done]
All repositories have been cleaned up.
Removing intermediate container 268efd201ed6
 ---> ca7b5a5163e4
Step 4/5 : RUN systemctl enable sshd
 ---> Running in fd6c26d9aedf
Removing intermediate container fd6c26d9aedf
 ---> ce86e9e8bcc0
Step 5/5 : RUN echo "root:root" | chpasswd
 ---> Running in aa7c897403aa
BAD PASSWORD: it is too short
BAD PASSWORD: is too simple
Removing intermediate container aa7c897403aa
 ---> 8ba71014bc58
Warning:  One or more build-args [GOARCH BINARY_REF] were not consumed
Successfully built 8ba71014bc58
Successfully tagged weaveworks/ignite-opensuse:leap-v0.10.0
docker tag weaveworks/ignite-opensuse:leap-v0.10.0 weaveworks/ignite-opensuse:leap
docker tag weaveworks/ignite-opensuse:leap-v0.10.0 weaveworks/ignite-opensuse:latest
make[1]: Leaving directory '/home/runner/work/ignite/ignite/images'
make build WHAT=opensuse       RELEASE=tumbleweed
make[1]: Entering directory '/home/runner/work/ignite/ignite/images'
sed "s|DOCKERARCH|amd64|g;/QEMUARCH/d" opensuse/Dockerfile \
	| docker build \
		--build-arg RELEASE \
		--build-arg BINARY_REF \
		--build-arg GOARCH \
		-f -\
		-t weaveworks/ignite-opensuse:tumbleweed-v0.10.0 \
		opensuse
Sending build context to Docker daemon  2.607kB

Step 1/5 : ARG RELEASE
Step 2/5 : FROM opensuse/${RELEASE}
latest: Pulling from opensuse/tumbleweed
24ccc6da3032: Pulling fs layer
24ccc6da3032: Verifying Checksum
24ccc6da3032: Download complete
24ccc6da3032: Pull complete
Digest: sha256:447ba278f40595ef54c9d5254ffb9e086ff5ec03bfce3b00b08091b4f017df02
Status: Downloaded newer image for opensuse/tumbleweed:latest
 ---> eda51fccb823
Step 3/5 : RUN zypper -n install         iproute         iputils         openssh         net-tools         systemd-sysvinit         udev         sudo         wget &&     zypper clean --all
 ---> Running in 20b28a9afa46
Retrieving repository 'openSUSE-Tumbleweed-Non-Oss' metadata [..done]
Building repository 'openSUSE-Tumbleweed-Non-Oss' cache [....done]
Retrieving repository 'openSUSE-Tumbleweed-Oss' metadata [......done]
Building repository 'openSUSE-Tumbleweed-Oss' cache [....done]
Retrieving repository 'openSUSE-Tumbleweed-Update' metadata [..done]
Building repository 'openSUSE-Tumbleweed-Update' cache [....done]
Loading repository data...
Reading installed packages...
'iproute' not found in package names. Trying capabilities.
Resolving package dependencies...

The following 72 NEW packages are going to be installed:
  busybox busybox-gzip chkstat dbus-1 diffutils hostname iproute2 iputils kbd kbd-legacy kmod libapparmor1 libargon2-1 libaudit1 libblkid1 libcap-ng0 libcares2 libcbor0_8 libcrypt1 libcryptsetup12 libdbus-1-3 libdevmapper1_03 libeconf0 libedit0 libelf1 libexpat1 libfdisk1 libfido2-1 libfido2-udev libhidapi-hidraw0 libjson-c5 libkmod2 libmetalink3 libmnl0 libmount1 libpkgconf3 libseccomp2 libsepol2 libsmartcols1 libutempter0 libuuid1 libxtables12 login_defs net-tools openssh openssh-clients openssh-common openssh-server pam pam-config pam_unix perl-base permissions permissions-config pkgconf pkgconf-m4 pkgconf-pkg-config sudo suse-module-tools system-group-hardware system-group-kvm systemd systemd-default-settings systemd-default-settings-branding-SLE systemd-presets-branding-MicroOS systemd-presets-common-SUSE systemd-rpm-macros systemd-sysvinit sysuser-shadow udev util-linux wget

72 new packages to install.
Overall download size: 21.7 MiB. Already cached: 0 B. After the operation, additional 59.1 MiB will be used.
Continue? [y/n/v/...? shows all options] (y): y
Retrieving package chkstat-1550_20210518-28.14.x86_64 (1/72), 412.0 KiB (  1.2 MiB unpacked)
Retrieving: chkstat-1550_20210518-28.14.x86_64.rpm [.done]
Retrieving package diffutils-3.7-4.5.x86_64 (2/72), 216.6 KiB (441.3 KiB unpacked)
Retrieving: diffutils-3.7-4.5.x86_64.rpm [done]
Retrieving package hostname-3.23-2.6.x86_64 (3/72),  25.9 KiB ( 37.5 KiB unpacked)
Retrieving: hostname-3.23-2.6.x86_64.rpm [done]
Retrieving package kbd-legacy-2.4.0-2.1.noarch (4/72), 570.2 KiB (560.2 KiB unpacked)
Retrieving: kbd-legacy-2.4.0-2.1.noarch.rpm [done]
Retrieving package libapparmor1-3.0.1-8.1.x86_64 (5/72),  98.2 KiB ( 83.8 KiB unpacked)
Retrieving: libapparmor1-3.0.1-8.1.x86_64.rpm [.done (12.0 KiB/s)]
Retrieving package libargon2-1-0.0+git20190520.62358ba-1.6.x86_64 (6/72),  20.7 KiB ( 30.2 KiB unpacked)
Retrieving: libargon2-1-0.0+git20190520.62358ba-1.6.x86_64.rpm [done]
Retrieving package libaudit1-3.0.2-1.1.x86_64 (7/72),  97.2 KiB (114.8 KiB unpacked)
Retrieving: libaudit1-3.0.2-1.1.x86_64.rpm [done]
Retrieving package libblkid1-2.36.2-2.1.x86_64 (8/72), 222.4 KiB (207.0 KiB unpacked)
Retrieving: libblkid1-2.36.2-2.1.x86_64.rpm [done]
Retrieving package libcap-ng0-0.7.10-1.29.x86_64 (9/72),  29.4 KiB ( 52.4 KiB unpacked)
Retrieving: libcap-ng0-0.7.10-1.29.x86_64.rpm [done]
Retrieving package libcares2-1.17.1-1.4.x86_64 (10/72),  64.2 KiB ( 94.8 KiB unpacked)
Retrieving: libcares2-1.17.1-1.4.x86_64.rpm [.done (11.9 KiB/s)]
Retrieving package libcbor0_8-0.8.0-1.4.x86_64 (11/72),  56.3 KiB (101.5 KiB unpacked)
Retrieving: libcbor0_8-0.8.0-1.4.x86_64.rpm [done]
Retrieving package libcrypt1-4.4.19-1.13.x86_64 (12/72), 111.8 KiB (260.9 KiB unpacked)
Retrieving: libcrypt1-4.4.19-1.13.x86_64.rpm [done]
Retrieving package libdbus-1-3-1.12.20-5.4.x86_64 (13/72), 242.0 KiB (322.0 KiB unpacked)
Retrieving: libdbus-1-3-1.12.20-5.4.x86_64.rpm [done]
Retrieving package libdevmapper1_03-1.02.177-2.1.x86_64 (14/72), 241.0 KiB (361.8 KiB unpacked)
Retrieving: libdevmapper1_03-1.02.177-2.1.x86_64.rpm [.done (936.6 KiB/s)]
Retrieving package libeconf0-0.4.1+git20210709.cf671f2-1.1.x86_64 (15/72),  33.0 KiB ( 39.7 KiB unpacked)
Retrieving: libeconf0-0.4.1+git20210709.cf671f2-1.1.x86_64.rpm [done]
Retrieving package libedit0-3.1.snap20180525-4.5.x86_64 (16/72), 107.0 KiB (237.6 KiB unpacked)
Retrieving: libedit0-3.1.snap20180525-4.5.x86_64.rpm [done]
Retrieving package libelf1-0.185-1.1.x86_64 (17/72),  85.2 KiB (102.7 KiB unpacked)
Retrieving: libelf1-0.185-1.1.x86_64.rpm [done]
Retrieving package libexpat1-2.4.1-1.1.x86_64 (18/72), 110.6 KiB (206.3 KiB unpacked)
Retrieving: libexpat1-2.4.1-1.1.x86_64.rpm [done]
Retrieving package libfido2-udev-1.7.0-1.2.noarch (19/72),  15.4 KiB ( 10.2 KiB unpacked)
Retrieving: libfido2-udev-1.7.0-1.2.noarch.rpm [done]
Retrieving package libhidapi-hidraw0-0.10.1-2.1.x86_64 (20/72),  29.9 KiB ( 59.2 KiB unpacked)
Retrieving: libhidapi-hidraw0-0.10.1-2.1.x86_64.rpm [.done (5.1 KiB/s)]
Retrieving package libjson-c5-0.15-2.2.x86_64 (21/72),  47.6 KiB ( 89.3 KiB unpacked)
Retrieving: libjson-c5-0.15-2.2.x86_64.rpm [done]
Retrieving package libkmod2-29-2.1.x86_64 (22/72),  68.3 KiB (102.2 KiB unpacked)
Retrieving: libkmod2-29-2.1.x86_64.rpm [done]
Retrieving package libmnl0-1.0.4-2.6.x86_64 (23/72),  26.5 KiB ( 52.4 KiB unpacked)
Retrieving: libmnl0-1.0.4-2.6.x86_64.rpm [done]
Retrieving package libpkgconf3-1.7.4-1.1.x86_64 (24/72),  38.9 KiB ( 68.9 KiB unpacked)
Retrieving: libpkgconf3-1.7.4-1.1.x86_64.rpm [done]
Retrieving package libseccomp2-2.5.1-1.1.x86_64 (25/72),  68.2 KiB (150.3 KiB unpacked)
Retrieving: libseccomp2-2.5.1-1.1.x86_64.rpm [.done (12.0 KiB/s)]
Retrieving package libsepol2-3.2-2.1.x86_64 (26/72), 283.2 KiB (713.4 KiB unpacked)
Retrieving: libsepol2-3.2-2.1.x86_64.rpm [done]
Retrieving package libsmartcols1-2.36.2-2.1.x86_64 (27/72), 176.0 KiB ( 98.2 KiB unpacked)
Retrieving: libsmartcols1-2.36.2-2.1.x86_64.rpm [done]
Retrieving package libuuid1-2.36.2-2.1.x86_64 (28/72), 150.2 KiB ( 30.2 KiB unpacked)
Retrieving: libuuid1-2.36.2-2.1.x86_64.rpm [done]
Retrieving package libxtables12-1.8.7-1.5.x86_64 (29/72),  56.3 KiB ( 59.0 KiB unpacked)
Retrieving: libxtables12-1.8.7-1.5.x86_64.rpm [done]
Retrieving package login_defs-4.8.1-7.1.noarch (30/72),  34.2 KiB ( 16.8 KiB unpacked)
Retrieving: login_defs-4.8.1-7.1.noarch.rpm [.done (930 B/s)]
Retrieving package openssh-common-8.4p1-6.1.x86_64 (31/72), 487.5 KiB (  1.4 MiB unpacked)
Retrieving: openssh-common-8.4p1-6.1.x86_64.rpm [done]
Retrieving package pkgconf-m4-1.7.4-1.1.noarch (32/72),  19.4 KiB ( 13.9 KiB unpacked)
Retrieving: pkgconf-m4-1.7.4-1.1.noarch.rpm [done]
Retrieving package systemd-default-settings-branding-SLE-0.7-1.2.noarch (33/72),  10.2 KiB (  1.5 KiB unpacked)
Retrieving: systemd-default-settings-branding-SLE-0.7-1.2.noarch.rpm [done]
Retrieving package systemd-presets-common-SUSE-15-19.1.noarch (34/72),  14.1 KiB (  4.3 KiB unpacked)
Retrieving: systemd-presets-common-SUSE-15-19.1.noarch.rpm [done]
Retrieving package systemd-rpm-macros-13-1.1.noarch (35/72),  18.6 KiB (  9.2 KiB unpacked)
Retrieving: systemd-rpm-macros-13-1.1.noarch.rpm [.done (12.1 KiB/s)]
Retrieving package permissions-config-1550_20210518-28.14.x86_64 (36/72),  59.4 KiB ( 46.9 KiB unpacked)
Retrieving: permissions-config-1550_20210518-28.14.x86_64.rpm [done]
Retrieving package net-tools-2.10-1.6.x86_64 (37/72),  58.0 KiB (120.4 KiB unpacked)
Retrieving: net-tools-2.10-1.6.x86_64.rpm [done]
Retrieving package libmount1-2.36.2-2.1.x86_64 (38/72), 243.8 KiB (262.7 KiB unpacked)
Retrieving: libmount1-2.36.2-2.1.x86_64.rpm [done]
Retrieving package perl-base-5.32.1-1.12.x86_64 (39/72),   1.5 MiB (  5.6 MiB unpacked)
Retrieving: perl-base-5.32.1-1.12.x86_64.rpm [.done (8.5 MiB/s)]
Retrieving package libmetalink3-0.1.3-3.12.x86_64 (40/72),  29.6 KiB ( 72.1 KiB unpacked)
Retrieving: libmetalink3-0.1.3-3.12.x86_64.rpm [done]
Retrieving package libfido2-1-1.7.0-1.2.x86_64 (41/72),  85.3 KiB (195.0 KiB unpacked)
Retrieving: libfido2-1-1.7.0-1.2.x86_64.rpm [done]
Retrieving package pkgconf-1.7.4-1.1.x86_64 (42/72),  43.8 KiB ( 76.3 KiB unpacked)
Retrieving: pkgconf-1.7.4-1.1.x86_64.rpm [done]
Retrieving package busybox-1.33.1-1.3.x86_64 (43/72), 643.5 KiB (  1.2 MiB unpacked)
Retrieving: busybox-1.33.1-1.3.x86_64.rpm [done]
Retrieving package libfdisk1-2.36.2-2.1.x86_64 (44/72), 264.3 KiB (322.5 KiB unpacked)
Retrieving: libfdisk1-2.36.2-2.1.x86_64.rpm [.done (5.1 KiB/s)]
Retrieving package libcryptsetup12-2.3.6-1.1.x86_64 (45/72), 268.6 KiB (473.6 KiB unpacked)
Retrieving: libcryptsetup12-2.3.6-1.1.x86_64.rpm [done]
Retrieving package iproute2-5.12-1.3.x86_64 (46/72),   1.0 MiB (  2.6 MiB unpacked)
Retrieving: iproute2-5.12-1.3.x86_64.rpm [done]
Retrieving package systemd-default-settings-0.7-1.2.noarch (47/72),  11.3 KiB (  2.0 KiB unpacked)
Retrieving: systemd-default-settings-0.7-1.2.noarch.rpm [done]
Retrieving package systemd-presets-branding-MicroOS-20200323-2.4.noarch (48/72),  10.6 KiB (  695   B unpacked)
Retrieving: systemd-presets-branding-MicroOS-20200323-2.4.noarch.rpm [done]
Retrieving package permissions-20210518.1550-28.14.x86_64 (49/72),  23.6 KiB (    0   B unpacked)
Retrieving: permissions-20210518.1550-28.14.x86_64.rpm [.done (919 B/s)]
Retrieving package wget-1.21.1-2.1.x86_64 (50/72), 398.2 KiB (765.0 KiB unpacked)
Retrieving: wget-1.21.1-2.1.x86_64.rpm [done]
Retrieving package openssh-clients-8.4p1-6.1.x86_64 (51/72), 789.5 KiB (  2.1 MiB unpacked)
Retrieving: openssh-clients-8.4p1-6.1.x86_64.rpm [done]
Retrieving package pkgconf-pkg-config-1.7.4-1.1.x86_64 (52/72),  14.7 KiB (  894   B unpacked)
Retrieving: pkgconf-pkg-config-1.7.4-1.1.x86_64.rpm [done]
Retrieving package busybox-gzip-1.33.1-18.1.noarch (53/72),  14.2 KiB ( 11.6 KiB unpacked)
Retrieving: busybox-gzip-1.33.1-18.1.noarch.rpm [.done (3.7 KiB/s)]
Retrieving package pam_unix-1.5.1-7.1.x86_64 (54/72),  68.9 KiB ( 54.8 KiB unpacked)
Retrieving: pam_unix-1.5.1-7.1.x86_64.rpm [done]
Retrieving package iputils-20210202-2.3.x86_64 (55/72), 122.1 KiB (333.6 KiB unpacked)
Retrieving: iputils-20210202-2.3.x86_64.rpm [done]
Retrieving package suse-module-tools-16.0.6-1.1.x86_64 (56/72),  46.1 KiB ( 80.5 KiB unpacked)
Retrieving: suse-module-tools-16.0.6-1.1.x86_64.rpm [done]
Retrieving package pam-1.5.1-7.1.x86_64 (57/72), 432.9 KiB (  1.6 MiB unpacked)
Retrieving: pam-1.5.1-7.1.x86_64.rpm [done]
Retrieving package kmod-29-2.1.x86_64 (58/72), 102.9 KiB (175.5 KiB unpacked)
Retrieving: kmod-29-2.1.x86_64.rpm [.done (2.3 KiB/s)]
Retrieving package sudo-1.9.7-1.2.x86_64 (59/72),   1.6 MiB (  6.2 MiB unpacked)
Retrieving: sudo-1.9.7-1.2.x86_64.rpm [done]
Retrieving package pam-config-1.3-2.1.x86_64 (60/72), 140.4 KiB (642.5 KiB unpacked)
Retrieving: pam-config-1.3-2.1.x86_64.rpm [done]
Retrieving package kbd-2.4.0-2.1.x86_64 (61/72),   1.9 MiB (  4.1 MiB unpacked)
Retrieving: kbd-2.4.0-2.1.x86_64.rpm [.done (8.5 MiB/s)]
Retrieving package systemd-248.3-2.1.x86_64 (62/72),   3.8 MiB ( 11.7 MiB unpacked)
Retrieving: systemd-248.3-2.1.x86_64.rpm [done]
Retrieving package sysuser-shadow-3.1-2.1.noarch (63/72),  13.0 KiB (  2.3 KiB unpacked)
Retrieving: sysuser-shadow-3.1-2.1.noarch.rpm [done]
Retrieving package system-group-hardware-20170617-22.1.noarch (64/72),  13.3 KiB (  358   B unpacked)
Retrieving: system-group-hardware-20170617-22.1.noarch.rpm [.done (305.6 KiB/s)]
Retrieving package libutempter0-1.2.0-3.14.x86_64 (65/72),  26.1 KiB ( 54.8 KiB unpacked)
Retrieving: libutempter0-1.2.0-3.14.x86_64.rpm [done]
Retrieving package util-linux-2.36.2-2.1.x86_64 (66/72),   1.1 MiB (  3.9 MiB unpacked)
Retrieving: util-linux-2.36.2-2.1.x86_64.rpm [done]
Retrieving package dbus-1-1.12.20-5.4.x86_64 (67/72), 232.7 KiB (605.0 KiB unpacked)
Retrieving: dbus-1-1.12.20-5.4.x86_64.rpm [done]
Retrieving package systemd-sysvinit-248.3-2.1.x86_64 (68/72), 389.8 KiB (  4.8 KiB unpacked)
Retrieving: systemd-sysvinit-248.3-2.1.x86_64.rpm [done]
Retrieving package system-group-kvm-20170617-22.1.noarch (69/72),  12.6 KiB (   37   B unpacked)
Retrieving: system-group-kvm-20170617-22.1.noarch.rpm [.done (12.5 KiB/s)]
Retrieving package openssh-server-8.4p1-6.1.x86_64 (70/72), 547.8 KiB (908.1 KiB unpacked)
Retrieving: openssh-server-8.4p1-6.1.x86_64.rpm [done]
Retrieving package udev-248.3-2.1.x86_64 (71/72),   1.5 MiB (  7.8 MiB unpacked)
Retrieving: udev-248.3-2.1.x86_64.rpm [done]
Retrieving package openssh-8.4p1-6.1.x86_64 (72/72),  77.9 KiB (    0   B unpacked)
Retrieving: openssh-8.4p1-6.1.x86_64.rpm [done]

Checking for file conflicts: [..........done]
( 1/72) Installing: chkstat-1550_20210518-28.14.x86_64 [...........done]
( 2/72) Installing: diffutils-3.7-4.5.x86_64 [..........done]
( 3/72) Installing: hostname-3.23-2.6.x86_64 [......done]
( 4/72) Installing: kbd-legacy-2.4.0-2.1.noarch [............done]
( 5/72) Installing: libapparmor1-3.0.1-8.1.x86_64 [......done]
( 6/72) Installing: libargon2-1-0.0+git20190520.62358ba-1.6.x86_64 [....done]
( 7/72) Installing: libaudit1-3.0.2-1.1.x86_64 [.......done]
( 8/72) Installing: libblkid1-2.36.2-2.1.x86_64 [.........done]
( 9/72) Installing: libcap-ng0-0.7.10-1.29.x86_64 [.....done]
(10/72) Installing: libcares2-1.17.1-1.4.x86_64 [......done]
(11/72) Installing: libcbor0_8-0.8.0-1.4.x86_64 [......done]
(12/72) Installing: libcrypt1-4.4.19-1.13.x86_64 [...........done]
(13/72) Installing: libdbus-1-3-1.12.20-5.4.x86_64 [........done]
(14/72) Installing: libdevmapper1_03-1.02.177-2.1.x86_64 [.........done]
(15/72) Installing: libeconf0-0.4.1+git20210709.cf671f2-1.1.x86_64 [.....done]
(16/72) Installing: libedit0-3.1.snap20180525-4.5.x86_64 [..........done]
(17/72) Installing: libelf1-0.185-1.1.x86_64 [......done]
(18/72) Installing: libexpat1-2.4.1-1.1.x86_64 [.........done]
(19/72) Installing: libfido2-udev-1.7.0-1.2.noarch [....done]
(20/72) Installing: libhidapi-hidraw0-0.10.1-2.1.x86_64 [.....done]
(21/72) Installing: libjson-c5-0.15-2.2.x86_64 [......done]
(22/72) Installing: libkmod2-29-2.1.x86_64 [......done]
(23/72) Installing: libmnl0-1.0.4-2.6.x86_64 [.....done]
(24/72) Installing: libpkgconf3-1.7.4-1.1.x86_64 [.....done]
(25/72) Installing: libseccomp2-2.5.1-1.1.x86_64 [........done]
(26/72) Installing: libsepol2-3.2-2.1.x86_64 [...........done]
(27/72) Installing: libsmartcols1-2.36.2-2.1.x86_64 [......done]
(28/72) Installing: libuuid1-2.36.2-2.1.x86_64 [....done]
(29/72) Installing: libxtables12-1.8.7-1.5.x86_64 [.....done]
(30/72) Installing: login_defs-4.8.1-7.1.noarch [....done]
(31/72) Installing: openssh-common-8.4p1-6.1.x86_64 [...........done]
(32/72) Installing: pkgconf-m4-1.7.4-1.1.noarch [....done]
(33/72) Installing: systemd-default-settings-branding-SLE-0.7-1.2.noarch [........done]
(34/72) Installing: systemd-presets-common-SUSE-15-19.1.noarch [......done]
(35/72) Installing: systemd-rpm-macros-13-1.1.noarch [....done]
(36/72) Installing: permissions-config-1550_20210518-28.14.x86_64 [.......done]
Additional rpm output:
Updating /etc/sysconfig/security ...
Checking permissions and ownerships - using the permissions files
	/usr/share/permissions/permissions
	/usr/share/permissions/permissions.easy
	/etc/permissions.local


(37/72) Installing: net-tools-2.10-1.6.x86_64 [.........done]
(38/72) Installing: libmount1-2.36.2-2.1.x86_64 [...........done]
(39/72) Installing: perl-base-5.32.1-1.12.x86_64 [............done]
(40/72) Installing: libmetalink3-0.1.3-3.12.x86_64 [......done]
(41/72) Installing: libfido2-1-1.7.0-1.2.x86_64 [.........done]
(42/72) Installing: pkgconf-1.7.4-1.1.x86_64 [.......done]
(43/72) Installing: busybox-1.33.1-1.3.x86_64 [...........done]
(44/72) Installing: libfdisk1-2.36.2-2.1.x86_64 [........done]
(45/72) Installing: libcryptsetup12-2.3.6-1.1.x86_64 [...........done]
(46/72) Installing: iproute2-5.12-1.3.x86_64 [............done]
(47/72) Installing: systemd-default-settings-0.7-1.2.noarch [.........done]
(48/72) Installing: systemd-presets-branding-MicroOS-20200323-2.4.noarch [.....done]
(49/72) Installing: permissions-20210518.1550-28.14.x86_64 [...done]
(50/72) Installing: wget-1.21.1-2.1.x86_64 [...........done]
(51/72) Installing: openssh-clients-8.4p1-6.1.x86_64 [............done]
(52/72) Installing: pkgconf-pkg-config-1.7.4-1.1.x86_64 [.....done]
(53/72) Installing: busybox-gzip-1.33.1-18.1.noarch [.......done]
(54/72) Installing: pam_unix-1.5.1-7.1.x86_64 [......done]
(55/72) Installing: iputils-20210202-2.3.x86_64 [..........done]
Additional rpm output:
/usr/bin/clockdiff: setting to root:root 0755 "cap_net_raw=p". (missing capabilities)


(56/72) Installing: suse-module-tools-16.0.6-1.1.x86_64 [.........done]
(57/72) Installing: pam-1.5.1-7.1.x86_64 [............done]
(58/72) Installing: kmod-29-2.1.x86_64 [.........done]
(59/72) Installing: sudo-1.9.7-1.2.x86_64 [..........done]
(60/72) Installing: pam-config-1.3-2.1.x86_64 [...........done]
Additional rpm output:
ERROR: module /lib64/security/pam_pwquality.so is not installed.
*** write_config (account, /etc/pam.d/common-account-pc, ...)
**** write config for pam_access.so (account, disabled)
**** write config for pam_unix2.so (account, disabled)
**** write config for pam_unix.so (account, enabled)
**** write config for pam_krb5.so (account, disabled)
**** write config for pam_localuser.so (account, disabled)
**** write config for pam_sss.so (account, disabled)
**** write config for pam_ldap.so (account, disabled)
**** write config for pam_nam.so (account, disabled)
**** write config for pam_winbind.so (account, disabled)
**** write config for pam_time.so (account, disabled)
*** write_config (auth, /etc/pam.d/common-auth-pc, ...)
**** write config for pam_env.so (auth, enabled)
**** write config for pam_group.so (auth, disabled)
**** write config for pam_pkcs11.so (auth, disabled)
**** write config for pam_fp.so (auth, disabled)
**** write config for pam_fprint.so (auth, disabled)
**** write config for pam_fprintd.so (auth, disabled)
**** write config for pam_thinkfinger.so (auth, disabled)
**** write config for pam_gnome_keyring.so (auth, disabled)
**** write config for pam_kwallet5.so (auth, disabled)
**** write config for pam_ssh.so (auth, disabled)
**** write config for pam_unix2.so (auth, disabled)
**** write config for pam_unix.so (auth, enabled)
**** write config for pam_ecryptfs.so (auth, disabled)
**** write config for pam_krb5.so (auth, disabled)
**** write config for pam_sss.so (auth, disabled)
**** write config for pam_ldap.so (auth, disabled)
**** write config for pam_nam.so (auth, disabled)
**** write config for pam_winbind.so (auth, disabled)
*** write_config (password, /etc/pam.d/common-password-pc, ...)
**** write config for pam_winbind.so (password, disabled)
**** write config for pam_pwcheck.so (password, disabled)
**** write config for pam_passwdqc.so (password, disabled)
**** write config for pam_pwquality.so (password, disabled)
**** write config for pam_cracklib.so (password, disabled)
**** write config for pam_pwhistory.so (password, disabled)
**** write config for pam_gnome_keyring.so (password, disabled)
**** write config for pam_kwallet5.so (password, disabled)
**** write config for pam_ecryptfs.so (password, disabled)
**** write config for pam_unix2.so (password, disabled)
**** write config for pam_unix.so (password, enabled)
**** write config for pam_make.so (password, disabled)
**** write config for pam_exec.so (password, disabled)
**** write config for pam_krb5.so (password, disabled)
**** write config for pam_sss.so (password, disabled)
**** write config for pam_ldap.so (password, disabled)
**** write config for pam_nam.so (password, disabled)
*** write_config (session, /etc/pam.d/common-session-pc, ...)
**** write config for pam_selinux.so (session, disabled)
**** write config for pam_mkhomedir.so (session, disabled)
**** write config for pam_systemd.so (session, disabled)
**** write config for pam_limits.so (session, enabled)
**** write config for pam_unix2.so (session, disabled)
**** write config for pam_unix.so (session, enabled)
**** write config for pam_apparmor.so (session, disabled)
**** write config for pam_krb5.so (session, disabled)
**** write config for pam_sss.so (session, disabled)
**** write config for pam_ldap.so (session, disabled)
**** write config for pam_winbind.so (session, disabled)
**** write config for pam_nam.so (session, disabled)
**** write config for pam_umask.so (session, enabled)
**** write config for pam_ssh.so (session, disabled)
**** write config for pam_selinux.so (session, disabled)
**** write config for pam_gnome_keyring.so (session, disabled)
**** write config for pam_kwallet5.so (session, disabled)
**** write config for pam_exec.so (session, disabled)
**** write config for pam_ecryptfs.so (session, disabled)
**** write config for pam_env.so (session, enabled)
**** write config for pam_mktemp.so (session, disabled)


(61/72) Installing: kbd-2.4.0-2.1.x86_64 [............done]
Additional rpm output:
Updating /etc/sysconfig/console ...
Updating /etc/sysconfig/keyboard ...


(62/72) Installing: systemd-248.3-2.1.x86_64 [............done]
Additional rpm output:
Creating group systemd-journal with gid 499.
Creating group systemd-network with gid 498.
Creating user systemd-network (systemd Network Management) with uid 498 and gid 498.
Creating group systemd-resolve with gid 497.
Creating user systemd-resolve (systemd Resolver) with uid 497 and gid 497.
Creating group systemd-timesync with gid 496.
Creating user systemd-timesync (systemd Time Synchronization) with uid 496 and gid 496.
Creating group systemd-coredump with gid 495.
Creating user systemd-coredump (systemd Core Dumper) with uid 495 and gid 495.
System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down
/usr/lib/tmpfiles.d/journal-nocow.conf:26: Failed to resolve specifier: uninitialized /etc detected, skipping
All rules containing unresolvable specifiers will be skipped.
/usr/lib/tmpfiles.d/static-nodes-permissions.conf:12: Failed to resolve group 'audio'.
/usr/lib/tmpfiles.d/static-nodes-permissions.conf:13: Failed to resolve group 'audio'.
/usr/lib/tmpfiles.d/static-nodes-permissions.conf:14: Failed to resolve group 'disk'.
/usr/lib/tmpfiles.d/static-nodes-permissions.conf:17: Failed to resolve group 'kvm'.
/usr/lib/tmpfiles.d/static-nodes-permissions.conf:18: Failed to resolve group 'kvm'.
/usr/lib/tmpfiles.d/static-nodes-permissions.conf:19: Failed to resolve group 'kvm'.
/usr/lib/tmpfiles.d/suse.conf:10: Failed to resolve group 'lock'.
/usr/lib/tmpfiles.d/suse.conf:13: Failed to resolve group 'utmp'.
/usr/lib/tmpfiles.d/suse.conf:14: Failed to resolve group 'utmp'.
/usr/lib/tmpfiles.d/suse.conf:15: Failed to resolve group 'utmp'.


(63/72) Installing: sysuser-shadow-3.1-2.1.noarch [....done]
(64/72) Installing: system-group-hardware-20170617-22.1.noarch [....done]
Additional rpm output:
/usr/bin/systemd-sysusers -
Creating group kmem with gid 494.
Creating group lock with gid 493.
Creating group tty with gid 5.
Creating group utmp with gid 492.
Creating group audio with gid 491.
Creating group cdrom with gid 490.
Creating group dialout with gid 489.
Creating group disk with gid 488.
Creating group input with gid 487.
Creating group lp with gid 486.
Creating group render with gid 485.
Creating group sgx with gid 484.
Creating group tape with gid 483.
Creating group video with gid 482.


(65/72) Installing: libutempter0-1.2.0-3.14.x86_64 [......done]
(66/72) Installing: util-linux-2.36.2-2.1.x86_64 [............done]
Additional rpm output:
/usr/bin/mount: setting to root:root 4755 (wrong permissions 0755)
/usr/bin/umount: setting to root:root 4755 (wrong permissions 0755)
/usr/bin/wall: setting to root:tty 2755 (wrong permissions 0755)
/usr/bin/write: setting to root:tty 2755 (wrong permissions 0755)
/usr/bin/su: setting to root:root 4755 (wrong permissions 0755)


(67/72) Installing: dbus-1-1.12.20-5.4.x86_64 [..........done]
Additional rpm output:
/usr/bin/systemd-sysusers -
Creating group messagebus with gid 481.
Creating user messagebus (User for D-Bus) with uid 481 and gid 481.
update-alternatives: using /usr/bin/dbus-launch.nox11 to provide /usr/bin/dbus-launch (dbus-launch) in auto mode


(68/72) Installing: systemd-sysvinit-248.3-2.1.x86_64 [........done]
(69/72) Installing: system-group-kvm-20170617-22.1.noarch [....done]
Additional rpm output:
/usr/bin/systemd-sysusers -
Creating group kvm with gid 36.


(70/72) Installing: openssh-server-8.4p1-6.1.x86_64 [...........done]
Additional rpm output:
/usr/bin/systemd-sysusers -
Creating group sshd with gid 480.
Creating user sshd (SSH daemon) with uid 480 and gid 480.
Updating /etc/sysconfig/ssh ...
Created symlink /etc/systemd/system/multi-user.target.wants/sshd.service -> /usr/lib/systemd/system/sshd.service.


(71/72) Installing: udev-248.3-2.1.x86_64 [............done]
(72/72) Installing: openssh-8.4p1-6.1.x86_64 [...done]
Executing %posttrans script 'login_defs-4.8.1-7.1.noarch.rpm' [...
Output of systemd-presets-common-SUSE-15-19.1.noarch.rpm %posttrans script:
    Created symlink /etc/systemd/system/default.target.wants/ca-certificates.path -> /usr/lib/systemd/system/ca-certificates.path.
    Created symlink /etc/systemd/system/getty.target.wants/getty@tty1.service -> /usr/lib/systemd/system/getty@.service.
    Created symlink /etc/systemd/system/multi-user.target.wants/remote-fs.target -> /usr/lib/systemd/system/remote-fs.target.
    Created symlink /etc/systemd/system/multi-user.target.wants/kbdsettings.service -> /usr/lib/systemd/system/kbdsettings.service.
    Created symlink /etc/systemd/user/basic.target.wants/systemd-tmpfiles-setup.service -> /usr/lib/systemd/user/systemd-tmpfiles-setup.service.
    Created symlink /etc/systemd/user/timers.target.wants/systemd-tmpfiles-clean.timer -> /usr/lib/systemd/user/systemd-tmpfiles-clean.timer.

..
Output of kmod-29-2.1.x86_64.rpm %posttrans script:
    ERROR: /usr/bin/dracut not found

Warning: kmod-29-2.1.x86_64.rpm %posttrans script failed (returned 1)
.
Output of kbd-2.4.0-2.1.x86_64.rpm %posttrans script:
    ERROR: /usr/bin/dracut not found

..
Output of udev-248.3-2.1.x86_64.rpm %posttrans script:
    ERROR: /usr/bin/dracut not found

Warning: udev-248.3-2.1.x86_64.rpm %posttrans script failed (returned 1)
..done]
All repositories have been cleaned up.
Removing intermediate container 20b28a9afa46
 ---> c9b6769c1c75
Step 4/5 : RUN systemctl enable sshd
 ---> Running in 85692713e651
Removing intermediate container 85692713e651
 ---> fe8733a72e96
Step 5/5 : RUN echo "root:root" | chpasswd
 ---> Running in 86557a46a4b1
/bin/sh: line 1: chpasswd: command not found
The command '/bin/sh -c echo "root:root" | chpasswd' returned a non-zero code: 127
make[1]: *** [Makefile:70: build] Error 127
make: *** [Makefile:174: build-opensuse] Error 2
make[1]: Leaving directory '/home/runner/work/ignite/ignite/images'
Error: Process completed with exit code 2.
```

</details>

--- a/images/opensuse/Dockerfile
+++ b/images/opensuse/Dockerfile
@@ -11,6 +11,7 @@ RUN zypper -n install \
         systemd-sysvinit \
         udev \
         sudo \
+        shadow \
         wget && \
     zypper clean --all
 

